lib/live_select/component.ex

```diff
@@ -615,7 +618,7 @@ defmodule LiveSelect.Component do
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 20 20"
       fill="currentColor"
-      class="w-5 h-5 @class"
+      class={["w-5 h-5", @class]}
```

Comment:
This fixes a small bug which prevented the `@class` interpolation.

Reply:
good catch!